settings_tab.py

```
@@ -0,0 +1,359 @@
+        s["ui_theme_mode"] = mode
+        s["ui_theme_name"] = theme_name
+        s["ui_font_family"] = family
+        s["ui_font_size"] = size
+
+        # size == 0 means "system default" — apply_fonts is a no-op for 0, so we
+        # explicitly restore the shipped named-font values before applying the
+        # theme. Otherwise the previously-applied custom size sticks.
+        if size <= 0 and not family:
+            try:
+                ui_theme.reset_fonts_to_system(self.root)
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)